dbt/models/default/columns.md

```diff
@@ -0,0 +1,384 @@
+- `232 - DRIVEWAYS - PATIOS`
+- `233 - FENCING`
+- `234 - GARAGE-BARN-BUTLER BUILDING`
+- `235 - OTHER-MINOR NEW CONSTRUCTION`
+- `236 - ???`
+- `251 - PARTITIONING`
+- `252 - DECONV/CONV AMT. LIV. UT.`
+- `253 - AIR CONDITIONING - FIREPLACES`
+- `254 - BASEMENT-ATTIC ROOM BATH`
+- `255 - REPLACE-REPAIR BUIDLING - FACTORY`
+- `256 - OTHER REMODELING`
+- `257 - REMODELING - INTERIOR ONLY`
+- `258 - REMODELING - EXTERIOR ONLY`
+- `271 - MAJOR IMPROVEMENT - WRECK`
+- `272 - MINOR IMPROVEMENT - WRECK`
+- `291 - MAJOR IMPROVEMENT - BURNOUT`
+- `292 - MINOR IMPROVEMENT - BURNOUT`
+- `300 - RAILROAD PROPERTIES`
+- `310 - NEW CONST MAJOR IMPROVEMENT`
+- `311 - CARRIER - NEW MAJOR CONSTRUCTION`
+- `312 - NON-CARRIER NEW MAJOR CONSTRUCTION`
+- `330 - NEW CONSTRUCTION  MINOR IMPROVEMENT`
+- `331 - CARRIER - REMODELING`
+- `332 - NON-CARRIER-REMODELING`
+- `350 - REMODELING`
+- `351 - CARRIER - REMODELING`
+- `352 - NON-CARRIER - REMODELING`
+- `370 - WRECKS`
+- `371 - MAJOR IMPROVEMENT WRECK`
+- `372 - MINOR IMPROVEMENT - WRECK`
+- `390 - BURNOUTS`
+- `391 - MAJOR IMPROVEMENT - BURNOUT`
+- `392 - MINOR IMPROVEMENT - BURNOUT`
+- `400 - EXEMPT PROPERTIES`
+- `411 - LEASEHOLD`
+- `510 - RECHECKS`
+- `511 - OCCUPANCY`
+- `512 - PRIOR YEAR PARTIAL ASSESSMENT`
+- `515 - MANUALLY ISSUED PERMIT`
+- `516 - PRIOR YEAR RECHECK`
+- `900 - OTHER`
+{% enddocs %}
+
+## permit_job_code_primary
+
+{% docs column_permit_job_code_primary %}
+Primary job description.
+
+This field functions as a permit classification field. The
+`job_code_secondary` and `improvement_code_*` fields provide
+codes with more details on the type of job that the permit represents.
+
+Possible values for this variable are:
+
+- null: No job description assigned
+- `1: RESIDENTIAL PERMIT`
+- `2: COMMERCIAL PERMIT`
+- `3: RAILROAD PERMIT`
+- `4: EXEMPT PERMIT`
+- `5: OFFICE PERMIT`
+- `6: OCCUPANCY PERMIT`
+- `7: OTHER`
+{% enddocs %}
+
+## permit_job_code_secondary
+
+{% docs column_permit_job_code_secondary %}
+Secondary job description.
+
+In conjunction with the `job_code_primary` and `improvement_code_*`
+fields, this field provides details on the type of job that the
+permit represents.
+
+Possible values for this variable are:
+
+- null: No job description assigned
+- `111 - NEW BUILDING`
+- `112 - ADDITIONS`
+- `113 - DORMERS`
+- `114 - OTHER - MAJOR NEW CONSTRUCTION`
+- `114.1 - OTHER - MAJOR NEW CONSTRUCTION - foundations`
+- `114.2 - OTHER - MAJOR NEW CONSTRUCTION - mobile home cement pads`
+- `114.3 - OTHER - MAJOR NEW CONSTRUCTION - structural changes`
+- `114.4 - OTHER - MAJOR NEW CONSTRUCTION - seasonal rooms that are conventionally heated (forced hot air, radiant, etc.) such as Florida Rooms or Sun Rooms.`
+- `114.5 - OTHER - MAJOR NEW CONSTRUCTION`
+- `114.6 - OTHER - MAJOR NEW CONSTRUCTION`
+- `114.7 - OTHER - MAJOR NEW CONSTRUCTION`
+- `114.8 - OTHER - MAJOR NEW CONSTRUCTION`
+- `114.9 - OTHER - MAJOR NEW CONSTRUCTION`
+- `131 - SWIMING POOL - TENNIS COURT`
```

Comment:
**nitpick:** I assume that the code is this way in the actual enumeration. If so, keep it as-is, if not, let's correct:

```suggestion
- `131 - SWIMMING POOL - TENNIS COURT`
```

Reply:
Good eye! I double-checked and these values are indeed misspelled in the actual enumeration, so I'm going to leave them as-is 😅

Reply:
On second thought, I decided that since we were going to clean up some of these values in 7d94c97522a10d4303a21f3e5634ae39c567aa4b, I might as will fix this one too.